docs/src/toml.md

```diff
@@ -35,6 +36,19 @@ type = "float"
 description = "Molecular weight dry air (kg/mol)"
 ```
 
+### Properly tagged parameter
+The parameter below has tags of varying specificity, from the package name to the specific parameter set it is intended for.
+
+```TOML
+[prandtl_number_0_grachev]
+alias = "Pr_0_Grachev"
+value = 0.98
+type = "float"
+description = "Pr_0 for Grachev universal functions. From Grachev et al, 2007. DOI: 10.1007/s10546-007-9177-6"
+tags = ["Grachev", "SurfaceFluxes", "UniversalFunctions"]
+```
+These can be searched via `get_tagged_parameter_names` and `get_tagged_parameter_values`.
```

Comment:
Perhaps be more explicit here, 

"If this convention is followed, to obtain the parameters used to build tagged by "Grachev" struct, one could call for example:
```julia 
# given toml_dict parsed from parameters.toml
grachev_params = get_tagged_parameter_values(toml_dict, "Grachev")
```